Estefany dark

StefaRueRome opened the pull request:
Opción para cambio de tema terminada, incluido los espacios de video, información del titulo y los primeros 3 tags de cada video en cada espacio de video.

Commit:
Test commit

Commit:
Content placeholders dark mode

Commit:
Test

Pruebas en la funcionalidad del cambio de tema

Commit:
Theme change done

Commit:
Information video placeholders done

PedroChaparro commented on src/index.html:
```diff
@@ -19,17 +19,18 @@
+					<div class="theme-icon" id="change-theme">
 						<img
-							src="/icons/moon-icon.svg"
+							src="/src/public/icons/moon-icon.svg"
```
Al menos en mi caso, las nuevas rutas de las imágenes no funcionan, ni iniciando el servidor de desarrollo con vite ni abriendo el HTML directamente con el navegador.

Estas rutas deberían comenzar por: `/images/1.png `o `/icons/icon.svg`, vite sabe que debe buscarlas en la carpeta public.

SilviaPabon replied:
Confirmo lo que el compañero Pedro comenta.
![image](https://user-images.githubusercontent.com/62608580/190522591-f45bdd7e-c526-4100-b9f8-53326cea3d33.png)
Pensé que sólo era a mí o que algo estaba haciendo mal, pero cambié las rutas como él lo comentó y el problema se solucionó, como se puede ver en la imagen superior.

PedroChaparro replied:
Arreglado.

PedroChaparro commented on src/style.css:
```diff
@@ -128,23 +131,24 @@ body {
 .video-placeholder__tag {
```
Los estilos de las cards con contenido de vídeos deberían usar una clase diferente ya que **la clase `.video-placeholder` y sus elementos (`.video-placeholder__tag`...) son solo para mostrar al usuario mientras se "trae" el contenido desde el back-end.**

PedroChaparro commented on src/style.css:
```diff
@@ -187,3 +198,42 @@ body {
 }
+
+/*Variables*/
+.theme-dark {
```
Veo que no son los colores de [figma](https://www.figma.com/file/USr5NT6kA7N6nN3ySMIlwC/ALAKO-PI202202?node-id=0%3A1). Personalmente no me molestan los colores usados, pero **ya que se aprobó el diseño con los otros colores creo que deberíamos respetarlo.**

PedroChaparro commented on src/index.html:
```diff
@@ -49,61 +50,131 @@ <h1 class="header__title" translate="no">ALAKO</h1>
+						<img src="/src/public/images/1.png" alt="video-1">
+					</div>
+					<div class="video-placeholder__title">
+						<label>Cómo usar gimp en 21 minutos edición de imágenes guía para principiantes como photoshop</label>
```
El elemento `label` de HTML está pensado para usarse en formularios. En su lugar propongo usar un `h2` o `h3` para el título de cada vídeo y un `p` o `span` para los tags del vídeo.

PedroChaparro commented on src/index.html:
```diff
@@ -49,61 +50,131 @@ <h1 class="header__title" translate="no">ALAKO</h1>
 		</main>
-		<script type="module" src="/main.js"></script>
+	<script>
+		var icon =document.getElementById("icon");
```
Buen trabajo con el cambio de icono y tema. Esto se puede pasar a un .js por fuera del HTML y cambiaría el `var` por un `const` en este caso. Nuevamente la ruta `/src/public` no me funciona, en un comentario anterior comenté la posible solución.

PedroChaparro reviewed (changes requested):
Revisar los comentarios individuales. En resúmen la clase  `.video-placeholder` no debería ser modificada para mostrar la información de los vídeos, se debería tener otra clase aparte para eso, lo demás son cosas más puntuales.

SilviaPabon reviewed (changes requested):
Me adhiero a los comentarios del compañero Pedro, para que se tengan en cuenta y se corrijan 👍🏻 . 

También quisiera comentar que conforme el diseño de Figma, faltaría realizar un hover, o un cambio de estilo cuando el mouse se pone encima de la card para que muestre el símbolito de play y haya un cambio como de opacidad en la imagen.

A mí no me disgustan los tags como están, y más teniendo presente que pueden haber tags largos, @PedroChaparro tu que tienes más experiencia en esa parte, el tamaño del background de los divs que tienen las etiquetas, ¿podría ajustarse de manera dinámica y es sencillo de implementar? para que de pronto @StefaRueRome lo tenga presente.

Por el resto bien, me gustó mucho lo del cambio de interfaz, genial!

PedroChaparro commented:
> ... @PedroChaparro tu que tienes más experiencia en esa parte, el tamaño del background de los divs que tienen las etiquetas, ¿podría ajustarse de manera dinámica y es sencillo de implementar? para que de pronto @StefaRueRome lo tenga presente.

Actualmente se está usando la clase `.video-placeholder__tag` la cual le pone un alto y ancho fijo al elemento (lo cual está bien para los placeholders). Si se usa otra clase el background se debería adaptar solo al ancho de la palabra y el alto de los caracteres.

StefaRueRome commented:
Muchas gracias por la realimentación dada, pero dichas correcciones no son impedimento para que se cumpla la US. Correcciones como declaración de variables, colores de figma, creación de un script aparte y demás cosas, finalmente cabe resaltar que en el sprint review se hizo la demostración del funcionamiento, pues el objetivo y resultado final era simplemente un cambio de colores claros a oscuros. Agradezco que se acepte la migración ya qué se está dilatando la realización de otras US por estas correcciones.

Obtener Outlook para iOS<https://aka.ms/o0ukef>
________________________________
De: Pedro Andrés Chaparro Quintero ***@***.***>
Enviado: Thursday, September 15, 2022 6:11:42 PM
Para: PedroChaparro/PI202202-alako-frontend ***@***.***>
Cc: Yurley Estefany Rueda Romero ***@***.***>; Mention ***@***.***>
Asunto: Re: [PedroChaparro/PI202202-alako-frontend] Estefany dark (PR #14)


A mí no me disgustan los tags como están, y más teniendo presente que pueden haber tags largos, @PedroChaparro<https://github.com/PedroChaparro> tu que tienes más experiencia en esa parte, el tamaño del background de los divs que tienen las etiquetas, ¿podría ajustarse de manera dinámica y es sencillo de implementar? para que de pronto @StefaRueRome<https://github.com/StefaRueRome> lo tenga presente.

Actualmente se está usando la clase .video-placeholder__tag lo cual le pone un alto y ancho fijo al elemento (lo cual está bien para los placeholders). Si se usa otra clase el background se debería adaptar solo al ancho de la palabra y el alto de los caracteres.

—
Reply to this email directly, view it on GitHub<https://github.com/PedroChaparro/PI202202-alako-frontend/pull/14#issuecomment-1248736264>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALKCNH6XGMJ3QN74MBZESALV6OUK5ANCNFSM6AAAAAAQNNATI4>.
You are receiving this because you were mentioned.Message ID: ***@***.***>

CONFIDENCIALIDAD: Este mensaje de correo electrónico contiene información privilegiada y confidencial; si usted no es el destinatario real del mismo, elimínelo de manera inmediata, reenvíelo a su remitente o informe de esta inconsistencia al correo ***@***.*** Así mismo, puede solicitar actualización de sus datos personales o la eliminación de nuestras listas de distribución a éste mismo correo. Para ver más sobre nuestra política de tratamiento y protección de datos personales ingrese al sitio https://www.upb.edu.co/es/politica-de-privacidad-upb

PedroChaparro commented:
> Muchas gracias por la realimentación dada, pero dichas correcciones no son impedimento para que se cumpla la US. Correcciones como declaración de variables, colores de figma, creación de un script aparte y demás cosas, finalmente cabe resaltar que en el sprint review se hizo la demostración del funcionamiento, pues el objetivo y resultado final era simplemente un cambio de colores claros a oscuros. Agradezco que se acepte la migración ya qué se está dilatando la realización de otras US por estas correcciones. 

Las tareas que se están retrasando son, en parte, de @SilviaPabon y ella misma está de acuerdo con que se realicen las correcciones. Entiendo el punto de que algunas correcciones son mínimas y no afectan directamente el funcionamiento de la tarea (como la creación del script aparte) pero **son cosas que no toman mucho tiempo y hacen parte de las buenas prácticas.** Más sin embargo, hay cosas que **sí o sí hay que corregir, ya que directamente no funcionan o no están alineadas con los criterios de aceptación**, como lo son: 

1. Los enlaces de las imágenes no funcionan. Tampoco le funcionaron a @SilviaPabon. 
2. La clase `.video-placeholder` es solamente para mostrar esto, **se necesita otra clase para mostrar como tal las cards de los vídeos**: 
![imagen](https://user-images.githubusercontent.com/62714297/190661669-59aad7a4-2cd8-45a5-adb3-5a8315145d44.png)
3. En el criterio de aceptación se menciona que la app web debe ser desdarrollada conforme al diseño de UI, **por lo que si nos guiamos por ese criterio, sí se deberían cambiar los colores y lo que mencioné de los tags.**

Commit:
Requested corrections

Se realizaron las siguientes correcciones:
Cambio a colores establecidos en figma.
Se creó el script por aparte y cambio a const de la variable del script.
Cambio de rutas de iconos e imagen.
Cards de videos aparte de los ya creados.
Cambio de texto a h2.

PedroChaparro committed:
Change bg color and reorganize file

PedroChaparro committed:
Prettier usage on html file

PedroChaparro committed:
Delete wraper div on titles

PedroChaparro committed:
Fix tags

Commit:
Play button on video

Hover para el botón de play

Commit:
Hover video play

PedroChaparro committed:
Some options for responsive and different height images

PedroChaparro reviewed (approved):
Aplicados los cambios solicitados, buen trabajo.

SilviaPabon reviewed (approved):
👍🏻😃  @StefaRueRome!

Woynert reviewed (approved):
I approve
